docs/deployment/migration-guide.md

```diff
@@ -21,6 +21,8 @@
 
 * Since Kyuubi 1.11, the configuration `spark.sql.watchdog.forcedMaxOutputRows` provided by Kyuubi Spark extension is removed, consider using `kyuubi.operation.result.max.rows` instead. Note, the latter works without requirement of installing Kyuubi Spark extension.
 
+* Since Kyuubi 1.11, the configuration `kyuubi.session.engine.spark.initialize.sql` is now applied for every session in shared engines (USER/GROUP/SERVER), correcting previous behavior where it was only applied for the first session.
```

Comment:
should it be "the configuration ... **set by client**"? I think the server-side conf (present in `kyuubi-defaults.conf`) also works previously, right?